internal/testcli/executor.go

```diff
@@ -308,3 +317,45 @@ func generateKeys(t *testing.T, n int) ([]*keys.PrivateKey, keys.PublicKeys) {
 	}
 	return privs, pubs
 }
+
+func (e *Executor) CheckTxTestInvokeOutput(t *testing.T, scriptSize int) {
+	e.CheckNextLine(t, `Hash:\s+`)
+	e.CheckNextLine(t, `OnChain:\s+false`)
+	e.CheckNextLine(t, `ValidUntil:\s+\d+`)
+	e.CheckNextLine(t, `Signer:\s+\w+`)
+	e.CheckNextLine(t, `SystemFee:\s+(\d|\.)+`)
+	e.CheckNextLine(t, `NetworkFee:\s+(\d|\.)+`)
+	e.CheckNextLine(t, `Script:\s+\w+`)
+	e.CheckScriptDump(t, scriptSize)
+}
+
+func (e *Executor) CheckScriptDump(t *testing.T, scriptSize int) {
+	e.CheckNextLine(t, `INDEX\s+`)
+	for i := 0; i < scriptSize; i++ {
+		e.CheckNextLine(t, `\d+\s+\w+`)
+	}
+}
+
+func DeployContract(t *testing.T, e *Executor, inPath, configPath, wallet, address, pass string) util.Uint160 {
+	config.Version = "0.90.0-test" // Contracts are compiler and we want NEFs to not change from run to run.
```

Comment:
It was here before, but it's likely to be "Contracts are compiled".

Reply:
A typo, fixed.